utils/src/main/kotlin/Extensions.kt

```diff
@@ -261,6 +261,10 @@ fun String.percentEncode(): String =
         // "+" instead of "%20", so apply the proper mapping here afterwards ("+" in the original string is
         // encoded as "%2B").
         .replace("+", "%20")
+        // "*" is a reserved character in RFC 3986.
+        .replace("*", "%2A")
```

Comment:
The analogous replacement in `String.fileSystemEncode()` can be removed now.

Reply:
Good point, done.